openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -94,8 +94,8 @@ <h3 class="message__title">{% trans "We could not submit your assessment" %}</h3
 
                 <ul class="list list--actions">
                     <li class="list--actions__item">
-                        <button type="submit" class="peer-assessment--001__assessment__submit action action--submit is--disabled">
-                            <span class="copy">{{ submit_button_text }}</span>
+                        <button type="submit" class="peer-assessment--001__assessment__submit action action--submit" disabled>
```

Comment:
No more "is--disabled" class on the LMS side of things.

Also, I removed the extra span around button text. The a11y tests were failing due to color contrast when the spans were present and the button was disabled. Apparently the axe toolkit isn't very smart about examining children of disabled components (disabled components do not need to meet color contrast standards).